client/landing/stepper/declarative-flow/internals/index.tsx

```diff
@@ -20,6 +20,7 @@ import { useStartStepperPerformanceTracking } from '../../utils/performance-trac
 import { StepRoute, StepperLoader } from './components';
 import { Boot } from './components/boot';
 import { RedirectToStep } from './components/redirect-to-step';
+import { useFlowAnalyctis } from './hooks/use-flow-analyctis';
```

Comment:
there's a typo in the file name. It's currently `analyctis`, probably should be `analytics`.

Reply:
Fixed here [592a0dc](https://github.com/Automattic/wp-calypso/pull/93179/commits/592a0dc2761532c77633af8c0c4b319d272ecd38) thanks